qiskit/pulse/pulse_lib/discrete.py

```diff
@@ -195,13 +282,20 @@ def gaussian(duration: int, amp: complex, sigma: float, name: Optional[str] = No
 
 def gaussian_deriv(duration: int, amp: complex, sigma: float,
                    name: Optional[str] = None) -> 'SamplePulse':
-    r"""Generates unnormalized gaussian derivative `SamplePulse`.
+    r"""Generates unnormalized gaussian derivative :class:`~qiskit.pulse.SamplePulse`.
+
+    For :math:`A=` ``amp`` and :math:`\sigma=` ``sigma`` applies the `midpoint` sampling strategy
+    to generate a discrete pulse sampled from the continuous function:
 
-    Applies `midpoint` sampling strategy to generate discrete pulse from continuous function.
+    .. math::
+
+        f(x) = A\frac{2(x - \mu)}{\sigma^2}\exp\left(\left(\frac{x - \mu}{\sigma}\right)^2 \right)
```

Comment:
If the gaussian func has x**2/(2sigma), then this is
```suggestion
        f(x) = A\frac{(x - \mu)}{\sigma^2}\exp\left(\left(\frac{x - \mu}{\sigma}\right)^2 \right)
```